Automattic/woocommerce-payments
# Add Dispute Status Chip into the header of the Dispute Details page

- The status needs to appear in the Details page as the design states

- Improved tests for testing all combinations of dispute reasons and statuses

Fixes #2341 

#### Changes proposed in this Pull Request

Title: Add Dispute Status Chip into the header of the Dispute Details page

Description: Currently the Dispute status is displayed in the Dispute List, but when entering into a dispute to see the details, the status of the dispute is not displayed anymore.

Before:

![image](https://user-images.githubusercontent.com/2612426/125769933-54ea1fa9-9d25-4bf1-b34f-918c23c79f16.png)

After:

![Markup on 2021-07-15 at 12:01:55](https://user-images.githubusercontent.com/2612426/125769964-93135ccd-7d37-447c-b4ad-f50f122bd0d2.png)


#### Testing instructions

- Navigate into Disputes
- Click on a dispute (if none, generate one using [Stripe dispute cards](https://stripe.com/docs/testing#disputes).
- See the Status next to the Title.

-------------------
